assets/css/base/_base.scss

```diff
@@ -52,7 +52,7 @@ button {
   font-size: rem(24);
   font-weight: 400;
   height: rem(54);
-  line-height: (34 / 24);
+  line-height: (rem(34) / rem(24));
```

Comment:
This isn't necessary. The original code produces the same output. Why was this changed? (See: https://gist.github.com/greatislander/619125d11df96890f0184e623741e71d)

Reply:
I was going through the code changing all the px to rem. I will change it back

Reply:
@TeddTech Those aren't pixel units, though. It's a Sass numeric operator: https://sass-lang.com/documentation/operators/numeric

Line height should always be expressed in unitless values, so we don't want to use rems (your change still resulted in a unitless value, but it's not necessary to add units to the input values).